app/config/config.yml

```diff
@@ -26,6 +26,44 @@ framework:
     fragments:       ~
     http_method_override: true
 
+nelmio_security:
+    clickjacking:
+        paths:
+            '^/.*': DENY
+    external_redirects:
+        abort: true
+        log: true
+        whitelist:
+            - %saml_remote_idp_host%
+
+    csp:
+        report:
+            report-uri: [/nelmio/csp/report]
```

Comment:
`none` for object and `[self]` for the rest would be better indeed :stuck_out_tongue_winking_eye: If we need something different in dev, we can configure that there.

And good point about the CSP enpoint, the report uri currently used is the default nelmio provided one. Would like to have the linked logging version though.